pillar.example

```diff
@@ -23,11 +23,14 @@ apache:
     # Default value for AddDefaultCharset in RedHat configuration
     default_charset: 'UTF-8'
 
+    # Formula manipulates service states (i.e. apache running/reload/restart)
+    # by default. Set 'False' here to leave apache service state alone.
+    manage_service_states:
```

Comment:
Please set the default value here (It happens to be an unspoken rule in most `pillar.example` files.) and move it out of `apache:lookup` into `apache`. (No OS-specific value => not handled in `map.jinja` => not in `apache:lookup`)

i.e.

```sls
apache:
  # ...
  manage_service_states: True  # default
```

Reply:
Hi @alxwr apologies for missing this, Thanks, I will definitely take your PR comments on board and look at this today hopefully.